e2e/pods_test.go

```diff
@@ -54,3 +48,58 @@ func TestPodLifecycle(t *testing.T) {
 		t.Fatal(string(out))
 	}
 }
+
+func TestPodWithCSIDriver(t *testing.T) {
```

Comment:
what environment is this testing against?

Reply:
> what environment is this testing against?
./hack/aks.sh